index.html

```diff
@@ -180,7 +180,24 @@ <h2 class="connect">LET'S CONNECT</h2>
       </div>
       
     </div>
+
   </section>
 
+  <section class="contacts">
+    <h2 class="contact-me">Contact Me</h2> 
+    <p>If you have an application you <br> are interested in developing, a <br> feature that you need built or a <br> project that needs coding. I’d <br> love to help with it </p>
+    <form action="https://formspree.io/f/meqvlald" method="POST">
+      <label >
+        <input type="text" id="name" name="name" maxlength="30" placeholder="Name" required>
+      </label>
+      <label class="mail-input">
```

Comment:
[OPTIONAL]

- [ ]  A recommended CSS practice is to use the `class` attribute for multiple elements. It would be better to use an `id` attribute here unless this selector will be re-used 👍🏽